proto/viam/app/v1/app.proto

```diff
@@ -822,6 +826,18 @@ message FragmentHistoryEntry {
   AuthenticatorInfo edited_by = 4 [(tagger.v1.tags) = "bson:\"edited_by\" json:\"edited_by\""];
 }
 
+message FragmentRevision {
+  string revision = 1;
+  google.protobuf.Timestamp created_at = 2;
+}
+
+message FragmentTag {
+  string tag = 1;
+  string revision = 2;
+  google.protobuf.Timestamp created_at = 3;
+  google.protobuf.Timestamp updated_at = 4;
```

Comment:
UI is not using these fields in latest design; given there's some quirks around trying to store this data, let's ditch it from the response